src/utils/semver.js

```diff
@@ -47,3 +54,15 @@ export async function getCleanDependencyName([depName, range]) {
 
   return [`${depName}@${range}`, `${depName}@${depVer}`, isLatest];
 }
+
+export function addSASTWarning(dirPath) {
+  const packageJsonPath = path.join(dirPath, "package.json");
```

Comment:
Isn't it possible to pass the package.json already parsed with `manifest.js` ?
It would also simplify UT's